### ISSUE TITLE
Use .ino

### ISSUE THREAD

**Issue by kj7rrv** [simonmonk/programming_arduino#2]
`.pde` is old; use the `.ino` extension.

**Comment by slowtoaccept** [comment]
I agree. .pde is the file extension for sketches used by the Processing interpreter/compiler. https://processing.org/. I have to tediously convert all your examples to .ino files. Hope they work.

**Comment by kj7rrv** [comment]
For now, you could probably script that.

**Comment by slowtoaccept** [comment]
Yes, but I’d need to read the book first!
